calibre-plugin/plugin-defaults.ini

```diff
@@ -1222,6 +1222,28 @@ extracategories:Lord of the Rings
 ## Site dedicated to these categories/characters/ships
 extracategories:Harry Potter
 
+[www.masseffect2.in]
+## Site dedicated to this fandom.
+extracategories: Mass Effect
+
+## Ucoz.com engine, upon which MassEffect2.in is based, imposes an unspecified limit on request frequency.
+## Sources vary from `5 requests per second' to `2 requests per second for more than 10 per minute'.
+## With default settings, a several-hours IP ban may follow, so set it lifted.
+slow_down_sleep_time: 2
+
+## Whether to exclude editor signature from the bottom if chapter text.
+exclude_editor_signature: false
+
+## Stories on the site almost never have cover image.
+## May be adjusted in `personal.ini' on per-story basis.
+never_make_cover: true
+
+## Titles for ratings identified by 1- or 2-letter codes from `ERATING system'
+## (`система Р.Е.Й.Т.И.Н.Г.').  MassEffect2.in and some other sites adopted it,
+## but changed titles and update them occasionally.
+rating_titles: R=RESTRICTED (16+), E=EXEMPT (18+), I=ART HOUSE, T=To every, A=IN=Иной мир, Nn=Новый мир, G=О\, Господи!
```

Comment:
It's not your fault, but including UTF8 in defaults.ini requires a change to configurable.py for CLI and web service to work properly.  I've pushed a change in FanFicFare/master for that, you should be able to just merge it with your branch.